### PR TITLE
Failedtestcases

### PR DESCRIPTION
**Pull request by SebaMutuku**

### CODE REVIEW

File: opensrp-anc/src/main/assets/config/contact-summary.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-properties_file_name: "contact_summary"
+propertiesFileName: "contact_summary"
```
Comment (dubdabasoduba) [inline opensrp-anc/src/main/assets/config/contact-summary.yml:2]: Why are we changing the naming convention here?

File: opensrp-anc/src/main/java/org/smartregister/anc/library/domain/YamlConfig.java [hunk 2]
```diff
@@ -8,35 +8,35 @@
 public class YamlConfig {
 
     private String group;
-    private String sub_group;
+    private String subGroup;
```
Comment (dubdabasoduba) [inline opensrp-anc/src/main/java/org/smartregister/anc/library/domain/YamlConfig.java:11]: Why are we changing to camel case here?

File: opensrp-anc/src/main/java/org/smartregister/anc/library/domain/YamlConfig.java [hunk 3]
```diff
@@ -8,35 +8,35 @@
...
     private List<YamlConfigItem> fields;
-    private String test_results;
-    private String properties_file_name;
+    private String testResults;
+    private String propertiesFileName;
```
Comment (dubdabasoduba) [inline opensrp-anc/src/main/java/org/smartregister/anc/library/domain/YamlConfig.java:14]: Why are we changing to camel case here?

File: opensrp-anc/src/main/java/org/smartregister/anc/library/model/RegisterModel.java [hunk 4]
```diff
@@ -60,4 +61,9 @@ public JSONObject getFormAsJson(String formName, String entityId, String current
     public String getInitials() {
         return Utils.getUserInitials();
     }
+
+    @VisibleForTesting
+    protected com.vijay.jsonwizard.utils.FormUtils getFormUtils() {
+        return new com.vijay.jsonwizard.utils.FormUtils();
+    }
```
Comment (dubdabasoduba) [inline opensrp-anc/src/main/java/org/smartregister/anc/library/model/RegisterModel.java:68]: Can't we access this method directly? `com.vijay.jsonwizard.utils.FormUtils();`

File: opensrp-anc/src/main/java/org/smartregister/anc/library/task/FinalizeContactTask.java [hunk 5]
```diff
@@ -27,6 +27,14 @@ public FinalizeContactTask(Context context, ProfileContract.Presenter mProfilePr
         this.intent = intent;
     }
 
+    @Override
+    protected void onPreExecute() {
+        ((ContactSummaryFinishActivity) context).showProgressDialog(R.string.please_wait_message);
+        ((ContactSummaryFinishActivity) context).getProgressDialog().setMessage(
+                String.format(context.getString(R.string.finalizing_contact),
+                        intent.getExtras().getInt(ConstantsUtils.IntentKeyUtils.CONTACT_NO)) + " data");
```
Comment (dubdabasoduba) [inline opensrp-anc/src/main/java/org/smartregister/anc/library/task/FinalizeContactTask.java:35]: Use constant files for literal strings

**Review by dubdabasoduba — CHANGES_REQUESTED** [review]

File: opensrp-anc/src/main/assets/config/contact-summary.yml (hunk 1 above)
Comment (SebaMutuku) [reply opensrp-anc/src/main/assets/config/contact-summary.yml]: Reverted

File: opensrp-anc/src/main/java/org/smartregister/anc/library/domain/YamlConfig.java (hunk 3 above)
Comment (SebaMutuku) [reply opensrp-anc/src/main/java/org/smartregister/anc/library/domain/YamlConfig.java]: Reverted

File: opensrp-anc/src/main/java/org/smartregister/anc/library/model/RegisterModel.java (hunk 4 above)
Comment (SebaMutuku) [reply opensrp-anc/src/main/java/org/smartregister/anc/library/model/RegisterModel.java]: Updated

**Review by dubdabasoduba — APPROVED** [review]

**Pull request merged**